client/lib/initialize-labels-state/index.js

```diff
@@ -47,7 +47,7 @@ export default ( formData, labelsData, paperSize, storeOptions, paymentMethod, n
 				ignoreValidation: hasOriginAddress ? null : mapValues( formData.origin, () => true ),
 				selectNormalized: true,
 				normalizationInProgress: false,
-				allowChangeCountry: false,
+				allowedCountries: [ 'US', 'PR' ],
```

Comment:
What do you think about defining this on the PHP side? My thought is that if we wanted to change the countries based on other store settings (or account level in the future), its easier to do so on the WP side of the house.